libraries/src/main/java/com/baeldung/commons/lang3/application/Application.java

```diff
@@ -0,0 +1,8 @@
+package com.baeldung.commons.lang3.application;
+
+public class Application {

```

Comment:
We don't need that class, right?

Reply:
Actually, we don't. I just kept it just in case the reader want's to drop the library's utility classes in it, rather than in the unit tests.